src/ui/DynamicFacetSearch/DynamicFacetSearch.ts

```diff
@@ -35,12 +36,17 @@ export class DynamicFacetSearch {
     return this.facetSearchController.search(terms);
   }
 
+  private getDisplayValue(value: string) {
+    return FacetUtils.getDisplayValueFromValueCaption(value, this.facet.options.field as string, this.facet.options.valueCaption);
+  }
+
   private createValuesFromResponse(response: IFacetSearchResponse): IComboboxValue[] {
     return response.values.map((value, index) => {
       const facetValue = new DynamicFacetValue(
         {
           value: value.rawValue,
-          displayValue: value.displayValue,
+          // TODO: remove when https://coveord.atlassian.net/browse/SEARCHAPI-4958 is fixed
+          displayValue: this.getDisplayValue(value.displayValue),
```

Comment:
found an issue here and logged it https://coveord.atlassian.net/browse/SEARCHAPI-4958

Reply:
Not directly related, but what is the difference between the `rawValue` and the `displayValue` on the facet search results? When would these values differ outside of value captions?

Reply:
I don't think they would otherwise caption is the only reason
From the doc of displayValue:
> The custom facet value display name, as specified in the `captions` argument of the facet request.

And for rawValue:
>  The original facet value, as retrieved from the field in the index.